doc/rtd/development/index.rst

```diff
@@ -0,0 +1,48 @@
+Development
+***********
+
+``Cloud-init`` is an open source project that warmly welcomes community
+projects, contributions, suggestions, fixes and constructive feedback. If you
+would like to contribute to ``cloud-init``, this set of documentation will help
+orient you with our processes.
+
+-----
+
+Contributing
+============
+
+.. toctree::
+   :maxdepth: 1
+
+   contributing.rst
+   module_creation.rst
+   code_review.rst
+   dir_layout.rst
+
+Debugging
+=========
+
+.. toctree::
+   :maxdepth: 1
+
+   security.rst
```

Comment:
I don't think that the security page fits under the debugging header.

Reply:
I wasn't 100% sure where that one should live tbh - where would you suggest?

Reply:
Since this page describes what happens when a security issue is reported, and how to report one, I think this would make sense a sub-page of bug reporting. In this PR, the bug reporting page is under the "How to guides", however since reporting a bug is a way to contribute, it might make sense to add a link to the bug reporting page under contributing, with the security page as a subpage of it.

What do you think about that?

Reply:
That's a good point. Ideally the security page (like performance) should be a top level section so I'd prefer not to make it a sub-page if we can avoid it (else we risk "burying" it). It might be simplest/tidiest (for now) if I change the "debugging" header to "debugging and reporting", and add a link to the same bug reporting page (currently in how-tos) from the dev section. The section would then be more conceptually complete and coherent, at least until we come to review the actual content and change it all around again :D